analysis_options.yaml

```diff
@@ -0,0 +1,21 @@
+# This is the default analysis_options.yaml for all packages and examples
```

Comment:
Putting this at root level to have a default and shared analysis_options for all the 4 packages in this repository!

Reply:
yes good thinking 👍